arangod/IResearch/IResearchLinkCoordinator.cpp

```diff
@@ -83,6 +83,77 @@ IResearchLinkCoordinator::IResearchLinkCoordinator(
   _sparse = true;   // always sparse
 }
 
+Result IResearchLinkCoordinator::init(
+    velocypack::Slice definition,
+    IResearchLink::InitCallback const& initCallback) {
+  auto r = IResearchLink::init(definition, initCallback);
```

Comment:
Do we really need to continue initialization in case of  `IResearchLink::init` failure ?

Reply:
I think not?

Reply:
But why? We will return failure anyway.

Reply:
Fixed